frontend/packages/client/src/pages/Community.js

```diff
@@ -196,6 +199,27 @@ export default function Community() {
     members: activeTab === 'members',
   };
 
+  const wrapperRef = useRef();
+  const aboutRef = useRef();
+  const proposalRef = useRef();
+  const memberRef = useRef();
+
+  useEffect(() => {
+    const refMap = {
+      about: aboutRef,
+      proposals: proposalRef,
+      members: memberRef,
+    };
+    const el = refMap[activeTab].current;
+    if (!el) return;
+
+    const elRect = el.getBoundingClientRect();
+    const parentRect = wrapperRef.current.getBoundingClientRect();
+    const offsetLeft = elRect.left - parentRect.left;
+    setActiveWidth(el.offsetWidth);
+    setActiveLeft(offsetLeft);
+  }, [activeTab]);
```

Comment:
This looks really cool!! maybe you can add `windowWidth` from `const { width: windowWidth } = useWindowDimensions(); ` so when resizing the activeLeft and  activeLeft values are updated dynamically to prevent wrong positioning unit component is refreshed